### PR TITLE
Add examples on how to create and update senders

### PR DESCRIPTION
**Pull request by sztupy**
These will be needed for a full integration with our API

### CODE REVIEW

File: docs/SendersApi.md [hunk 1]
```diff
@@ -298,7 +298,7 @@ apiClient.setApiSecret("<secret>");
 apiClient.setBasePath("https://api-sandbox.bitpesa.co/v1");
 
 SendersApi apiInstance = new SendersApi(apiClient);
-Sender sender = {
+SenderRequest senderRequest = {
```
Comment (jara) [inline docs/SendersApi.md:301]: Shouldn't the most recent update to the Swagger docs fix this on it's own? I had this issue in the docs initially but saw it was fixed when I went to make this change.

Comment (sztupy) [reply docs/SendersApi.md:301]: Yes, the `main` directory changes are always regenerated, however the `test` directories are not overriden by the generator (and you can't force it unfortunately, unless you delete the directory first), and needs to be done manually.

Comment (sztupy) [reply docs/SendersApi.md:301]: There was also an issue with how we regenerated the projects - the `.travis.yml` in `api-specification` runs for not just master but staging as well, and the staging branch has an older version - which pushed up a wrong SDK into these projects the same time I also modified some files - and this made the `main` and `test` directories out of sync. These changes here fix this

Comment (sztupy) [reply docs/SendersApi.md:301]: I think @jjromeo also encountered issues around this

**Review by jara — APPROVED** [review]

**Pull request merged**